x-pack/solutions/security/plugins/security_solution/public/explore/network/components/embeddables/map_tool_tip/map_tool_tip.tsx

```diff
@@ -74,24 +76,28 @@ export const MapToolTipComponent = ({
 
     return (
       <div>
-        {featureGeometry != null && featureGeometry.type === 'LineString' ? (
-          <LineToolTipContent
-            contextId={`${features[featureIndex].layerId}-${features[featureIndex].id}-${featureIndex}`}
-            featureProps={featureProps}
-          />
-        ) : (
-          <PointToolTipContent
-            contextId={`${features[featureIndex].layerId}-${features[featureIndex].id}-${featureIndex}`}
-            featureProps={featureProps}
-          />
-        )}
         {features.length > 1 && (
-          <ToolTipFooter
-            featureIndex={featureIndex}
-            totalFeatures={features.length}
-            previousFeature={handlePreviousFeature}
-            nextFeature={handleNextFeature}
-          />
+          <>
+            {featureGeometry != null && featureGeometry.type === 'LineString' ? (
+              <LineToolTipContent
+                data-test-subj="line-tool-tip-content"
+                contextId={`${features[featureIndex].layerId}-${features[featureIndex].id}-${featureIndex}`}
+                featureProps={featureProps}
+              />
+            ) : (
+              <PointToolTipContent
+                data-test-subj="point-tool-tip-content"
+                contextId={`${features[featureIndex].layerId}-${features[featureIndex].id}-${featureIndex}`}
+                featureProps={featureProps}
+              />
+            )}
```

Comment:
<img width="2558" alt="Image" src="https://github.com/user-attachments/assets/f330cde9-3b2e-45b1-8785-491da821f06d" />

Render tooltip content only when features are available to avoid the error above.

Reply:
❤️ Thank you!